src/test/java/com/atlassian/bitbucket/jenkins/internal/trigger/register/PullRequestStoreImplTest.java

```diff
@@ -42,53 +36,59 @@ private BitbucketPullRequest setupPR(String newKey) {
 
     @Test
     public void testAddPRWithNewKey() {
-        BitbucketPullRequest bitbucketPullRequest = setupPR(key);
+        BitbucketPullRequest bitbucketPullRequest = setupPR(key, BitbucketPullState.OPEN, 1);
         PullRequestStoreImpl.CacheKey cacheKey = new PullRequestStoreImpl.CacheKey(key, slug, serverId);
```

Comment:
This `cacheKey` (and every other ocarance of `CacheKey` in this class) is unused and should be removed.